proto/toit/api/user.proto

```diff
@@ -83,3 +86,19 @@ message CreatePaymentSubscriptionRequest {
 
 message CreatePaymentSubscriptionResponse {
 }
+
+message ListPaymentInvoicesRequest {
+    int64 limit = 1;
```

Comment:
It should be possible to start a list operation from a specific offset (invoiceID) add a `bytes offset` parameter. and add a comment that if set with a invoiceId the list would start after that invoiceID.